src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -690,10 +690,11 @@ private void executeCrafting( final IEnergyGrid eg, final CraftingGridCache cc )
 											fuzz = fuzz.copy();
 											fuzz.setStackSize( input[x].getStackSize() );
 
-											if( details.isValidItemForSlot( x, fuzz.getItemStack(), this.getWorld() ) )
+											// TODO: check if OK
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)